static/js/modules/glossary.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+var terms = require('./terms');
+var _ = require('underscore');
+
+var glossaryLink = $('.term'),
+    glossaryIsOpen = false,
+    indexToLowercase,
+    findDefinition,
+    setDefinition,
+    showGlossary,
+    hideGlossary,
+    clearTerm;
+
+// Indexing the terms and then lowercasing the indices
+indexToLowercase = function(arrayOfObjects, index){
+    var key, 
+        newKey,
+        object = _.indexBy(arrayOfObjects, index),
```

Comment:
I would just say we should rename things like `object` and `newObject` since they don't describe the values within. Maye `terms` and `lowercaseTerms` respectively?